lib/pages/backdrop.dart

```diff
@@ -166,35 +86,18 @@ class _BackdropState extends State<Backdrop>
     }
   }
 
-  void toggleSettings() {
+  void _toggleSettings() {
     // Animate the settings panel to open or close.
-    widget.controller
-        .fling(velocity: widget.isSettingsOpenNotifier.value ? -1 : 1);
-    setState(() {
-      widget.isSettingsOpenNotifier.value =
-          !widget.isSettingsOpenNotifier.value;
-    });
-    // Animate the settings icon.
+    _settingsPanelController.fling(
+        velocity: _isSettingsOpenNotifier.value ? -1 : 1);
+    _isSettingsOpenNotifier.value = !_isSettingsOpenNotifier.value;
     initAnimationLayer();
     isActive.value = true;
   }
 
```

Comment:
nit: Code like `initAnimationLayer`, `isActive.value = true`, `FlutterActorArtboard`, `FlareAnimationLayer`, `bool advance(FlutterActorArtboard artboard, double elapsed)` , `FlareController` are all related to the FlareIcon.

Not sure if its possible to move it down to the `SettingsIcon` or does that affect the performance?

Reply:
I tried that but did not have any luck, but I made a future issue for it: https://github.com/flutter/gallery/issues/76